### PR TITLE
Get Slider return value bug fixed

### PR DESCRIPTION
**Pull request by turanheydarli**

### CODE REVIEW

**Comment by turanheydarli** [comment]
Get Slider return value bug fixed

**Comment by dmitry-ivashenko** [comment]
Hey! Thanks for your contribution. I see too many unnecessary files in the diff. Could you please leave only the valuable ones?

**Comment by turanheydarli** [comment]
Hey! You're welcome. Here is the cleaned-up version. Thanks for pointing that out.

File: Assets/Plugins/StatefulUI/Runtime/Core/StatefulComponent.cs.meta [hunk 1]
```diff
@@ -4,7 +4,7 @@ MonoImporter:
   externalObjects: {}
   serializedVersion: 2
   defaultReferences: []
-  executionOrder: 0
+  executionOrder: 8
```
Comment (dmitry-ivashenko) [inline Assets/Plugins/StatefulUI/Runtime/Core/StatefulComponent.cs.meta]: What is it for?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "displayName": "Stateful UI",
+    "description": "Stateful UI",
+    "unity": "2022.3",
+    "author": {
+      "name": "Dmitrii Ivashchenko",
+      "email": "ivaschenko.dmitrii@gmail.com",
+      "url": "https://bit.ly/dmitry_ivashenko"
+    }
+  }
+
```
Comment (dmitry-ivashenko) [inline package.json]: We already have the package.json inside Assets/Plugins/StatefulUI. 
You can install the library using UPM as described in README.
Please remove it.

**Review by dmitry-ivashenko — COMMENTED** [review]
@turanheydarli please check my comment

File: Assets/Plugins/StatefulUI/Runtime/Core/StatefulComponent.cs.meta (hunk 1 above)
Comment (turanheydarli) [reply Assets/Plugins/StatefulUI/Runtime/Core/StatefulComponent.cs.meta]: Hi, Dimitry! I got some errors when initializing my screen on OnEnable and I tried to change execution order for this issue. Actually it's project spesific error and not nessesary.

**Review by dmitry-ivashenko — APPROVED** [review]
That's okay now, thanks!

**Pull request merged**